test/testing.cmake

```diff
@@ -2,12 +2,36 @@ set(MESHES ""
     CACHE STRING
     "Extracted http://scorec.rpi.edu/pumi/pumi_test_meshes.tar.gz")
 function(mpi_test TESTNAME PROCS EXE)
+  if(SCOREC_NO_MPI)
+  if(${PROCS} EQUAL "1")
```

Comment:
```suggestion
if(SCOREC_NO_MPI AND ${PROCS} EQUAL "1")
```
Will that work?

Reply:
No, and I updated the indentation to make it clearer what's going on. If SCOREC_NO_MPI is given, I want to only add serial tests. If AND is used instead, then parallel tests will still be added as usual.